.devcontainer/spin/install.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+VERSION="v0.6.0"
```

Comment:
Should this use the Spin install script instead from https://developer.fermyon.com/spin/install?

Reply:
That's a really great point @radu-matei I will try that out.